.gitlab-ci.yml

```diff
@@ -57,5 +57,41 @@ deploy-python-tool-production:
     ENVIRONMENT_NAME: production
     EVA_SUBMISSION_PATH: $EVA_SUBMISSION_PRODUCTION_PATH
     DEPLOY_POINT: $CI_COMMIT_TAG
+  only:
+    - tags
+
+# Not executed, parent job definition
+.deploy-python-tool-codon:
+  stage: deploy
+  image: python:3.8.7-slim-buster
+  environment:
+    name: $ENVIRONMENT_NAME
+  script:
+    # Start ssh-agent and set identity then run deploy script
+    # See https://docs.gitlab.com/ee/ci/ssh_keys/
+    - apt-get update
+    - apt-get -y install openssh-client
+    - eval "$(ssh-agent)"
+    - echo "$EVA_ETL_PRIVATE_KEY" | tr -d '\r' | ssh-add -
+    - ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${CLUSTER_USER}@${CLUSTER_LOGIN_NODE_CODON} "${EVA_SUBMISSION_PATH}/deploy.sh ${DEPLOY_POINT}"
```

Comment:
I was wondering if we could take this opportunity to switch to using [EVA_SOFTWARE_PATH and ENVIRONMENT_NAME](https://github.com/EBIvariation/covid19dp-submission/blob/f61bcb871a32392cf8d8e43f2729d190e8c278ca/.gitlab-ci.yml#L40) ?